pandas/core/generic.py

```diff
@@ -9546,12 +9544,13 @@ def describe(
         ...   np.datetime64("2010-01-01")
         ... ])
         >>> s.describe()
-        count                       3
-        unique                      2
-        top       2010-01-01 00:00:00
-        freq                        2
-        first     2000-01-01 00:00:00
-        last      2010-01-01 00:00:00
+        count                      3
```

Comment:
are these on the doctest list that we check?

Reply:
Since 4d66fa8 they are.